src/main/java/com/teamabnormals/abnormals_core/common/network/MessageC2SUpdateSlabfishHat.java

```diff
@@ -0,0 +1,47 @@
+package com.teamabnormals.abnormals_core.common.network;
+
+import com.teamabnormals.abnormals_core.common.world.storage.tracking.IDataManager;
+import com.teamabnormals.abnormals_core.core.AbnormalsCore;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.network.PacketBuffer;
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.fml.network.NetworkEvent;
+
+import java.util.function.Supplier;
+
+/**
+ * Message for updating if the Slabfish hat is enabled.
```

Comment:
Isn't this description misleading? Doesn't it do more than just update if the Slabfish hat is enabled?

Reply:
Forgot to update it when I changed everything. :P